pkg/client/resmgmt/resmgmt_test.go

```diff
@@ -1612,6 +1612,42 @@ func TestGetConfigSignaturesFromIdentities(t *testing.T) {
 	assert.NotNil(t, signature, "signatures must not be empty")
 }
 
+func TestCheckRequiredCCProposalParams(t *testing.T) {
+	// Valid request
+	ccPolicy := cauthdsl.SignedByMspMember("Org1MSP")
+	req := InstantiateCCRequest{Name: "name", Version: "version", Path: "path", Policy: ccPolicy}
+
+	// Test empty channel lang
+	err := checkRequiredCCProposalParams("mychannel", &req)
+	if err != nil {
+		t.Fatal("Should have failed for empty request")
```

Comment:
Wrong message?

Reply:
Thanks, The wrong message has been deleted.